en/intro-concepts.md

```diff
@@ -13,18 +13,23 @@ MAAS, not to mention the rest of this documentation.
 ## Nodes
 
 A *node* is a general term that refers to multiple, more specific objects.
-Basically, it is a networked object that is known to MAAS.
+Nodes are managed by MAAS through a life cycle, from enlistment into MAAS,
+through commissioning, allocation and deployment. Nodes are then either
+released back into the pool of nodes or retired.
 
 Nodes include:
 
 - Controllers
 - Machines
 - Devices
 
+See [Node actions][node-actions] below for an overview of a node's life cycle.
+
 ### Controllers
 
 There are two types of controllers: a *region controller* and a *rack
-controller*.
+controller*. The region controller deals with operator requests while one or
+more rack controllers provide the high-bandwidth services to multiple racks.
```

Comment:
This sounds unclear to me at this time: "one or more rack controllers provide the high-bandwidth services to multiple racks" as the data centre "server rack" idea has yet to be mentioned.

Reply:
Thanks for the feedback @pmatulis! I'll make some changes.